### DISCUSSION TITLE
Future Ready Talent Deadline

### DISCUSSION THREAD

**Discussion by NapsterAD** [github/india#793] [Students]
I got an email to submit project synopsis before 28th of February . But after logging in into the FRT Portal it is showing deadline of 14 may 2022 . Please me the procedures ( how to start , self learning , sessions , synopsis and all ). Help me ASAP please !!!

**Comment by PRIYA-S-102** [comment]
have you complete the project in FRT?
if yes
can you please me in submitting ?

**Comment by ey-mentor1** [comment]
Hello, 
All the above requested steps are mentioned and explained in details in FRT portal.
Please refer that.
Thank You

**Reply by PRIYA-S-102** [reply]
I tried but can't understand

**Reply by ey-mentor1** [reply]
Hello,
All the details are elaborated in the FRT portal.
Please go through them
Also, visit our YouTube channel.
https://youtube.com/channel/UCRQbNl0J1ky2hxPoB3TVEMA 
Thank you

**Reply by PRIYA-S-102** [reply]
Hello,
I had seen a Common Project Mentoring Session Recording on Youtube.
In that, they had said two parallel Streams for getting a Certificate 
One is without submitting Project but Completing all the self mandatory courses  and 
The Other one is submitting the project without completing Self Mandatory Courses 
I have Completed all the self-learning courses 
Could you please tell me how to apply for the Internship certificate without credits?
Thank you.

**Reply by ey-mentor1** [reply]
Hello,
You have create Azure free trial account, in case you don't opt for student developer pack.
https://azure.microsoft.com/en-in/free/
Thank you

**Comment by PRIYA-S-102** [comment]
No Iam Asking that without submitting project can I get the internship
certificate without credits

On Wed, Apr 13, 2022, 11:14 PM ey-mentor1 ***@***.***> wrote:

> Hello,
> You have create Azure free trial account, in case you don't opt for
> student developer pack.
> https://azure.microsoft.com/en-in/free/
> Thank you
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/github/india/discussions/793#discussioncomment-2561581>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV35HRA3GKHGNV424PQITILVE4BW5ANCNFSM5PMPOGHQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

-- 

*
*
*
*

*S.A. ENGINEERING COLLEGE**
*
*(An Autonomous Institution 
affiliated to Anna University, Chennai)*
*(Accredited by NBA, NAAC with 'A' 
Grade & ISO 9001:2015 Certified Institution)*
*Phone: 044-26801999  | 
Email: ***@***.*** ***@***.***> | Website: www.saec.ac.in 
<https://www.saec.ac.in>*


*Disclaimer: *The content of this email is 
confidential and intended for the recipient specified in message only. It 
is strictly forbidden to share any part of this message with any third 
party, without a written consent of the sender. If you received this 
message by mistake, please reply to this message and follow with its 
deletion, so that we can ensure such a mistake does not occur in the 
future.

**Reply by ey-mentor1** [reply]
No

Please note, In order to pass the evaluation, the project should satisfy following points:
a) Detailed description and explaination with suffiecient screenshots in github readme. 
b) Atleast one Azure service should be used. Github readme should explain the use of that service in your project. 
c) The Project URL should also be added in Github readme. 
d) Video URL demonstrating project demo, Azure service used and deployment of code on any Azure service in Github readme is preferred.

**Comment by PRIYA-S-102** [comment]
Okay

On Thu, Apr 14, 2022, 9:22 AM ey-mentor1 ***@***.***> wrote:

> No
>
> Please note, In order to pass the evaluation, the project should satisfy
> following points:
> a) Detailed description and explaination with suffiecient screenshots in
> github readme.
> b) Atleast one Azure service should be used. Github readme should explain
> the use of that service in your project.
> c) The Project URL should also be added in Github readme.
> d) Video URL demonstrating project demo, Azure service used and deployment
> of code on any Azure service in Github readme is preferred.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/github/india/discussions/793#discussioncomment-2564139>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV35HRFW3SHDCRSIWRXMDR3VE6I7FANCNFSM5PMPOGHQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

-- 

*
*
*
*

*S.A. ENGINEERING COLLEGE**
*
*(An Autonomous Institution 
affiliated to Anna University, Chennai)*
*(Accredited by NBA, NAAC with 'A' 
Grade & ISO 9001:2015 Certified Institution)*
*Phone: 044-26801999  | 
Email: ***@***.*** ***@***.***> | Website: www.saec.ac.in 
<https://www.saec.ac.in>*


*Disclaimer: *The content of this email is 
confidential and intended for the recipient specified in message only. It 
is strictly forbidden to share any part of this message with any third 
party, without a written consent of the sender. If you received this 
message by mistake, please reply to this message and follow with its 
deletion, so that we can ensure such a mistake does not occur in the 
future.
